docs/ManualDeploymentExtended.md

```diff
@@ -122,17 +124,12 @@ Tor can be used by the following components to provide enhanced privacy and/or h
 - Bitcoin-core Daemon
 - Lightning Network Daemon (lnd).
 
+Additional information running Bitcoin Core with Tor support can be found [here](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md).
+
 ##### 🚚 Install
 
 ```bash
-~$ sudo apt install apt-transport-https
-~$ sudo vi /etc/apt/sources.list # (and append two lines below)
-deb https://deb.torproject.org/torproject.org bionic main
-deb-src https://deb.torproject.org/torproject.org bionic main
-~$ curl https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gpg --import
-~$ gpg --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | sudo apt-key add -
-~$ sudo apt update
-~$ sudo apt install tor deb.torproject.org-keyring
```

Comment:
Isn't this necessary anymore? Maybe it'd make sense to optionally include this for previous Ubuntu versions?

Reply:
I think that's good we just put things for the latest version if this is simpler.

Reply:
The recommendation on the Tor website is to use "apt install" (I can't recall the reason I didn't use that last time but most likely it was the Ubuntu 18.04 package not supporting Torv3).